src/PrestaShopBundle/Resources/config/services/bundle/event_listener.yml

```diff
@@ -23,6 +23,11 @@ services:
     tags:
       - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest }
 
+  PrestaShopBundle\EventListener\AuthorizationServerFeatureListener:
+    autowire: true
+    tags:
+      - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest, priority: 6 }
```

Comment:
I think the priority could be higher than the `ScopeCheckerListener` Even if it's 8 and it is run before the Firall listener that prefills the Security user correctly it doesn't matter Regardless of the logged status of the user the listener should block the request early based on the configuration anyway